### PR TITLE
docs(steps): fix internal lowercase

### PR DESCRIPTION
**Pull request by thinkasany**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

1. `@internal`通常用于内部的属性，所以是不希望对外展示的，这边出现了大小写错误，导致这个暴露出来了。
2. `finishIconBorderColor`  是由其他token控制的，并不支持直接配置使用
image

image

before：image
after: 
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      docs(steps): fix internal lowercase     |
| 🇨🇳 Chinese |    docs(steps):  修复小写internal 注解|

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/steps/style/index.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export interface ComponentToken {
    */
   finishIconBgColor: string;
   /**
-   * @Internal
+   * @internal
```
Comment (yoyo837) [inline components/steps/style/index.ts:102]: 还能有这样的问题...

**Review by yoyo837 — APPROVED** [review]

File: components/steps/style/index.ts (hunk 1 above)
Comment (thinkasany) [reply components/steps/style/index.ts:102]: 确实神奇，仅此一处出现大小写错误，还奇怪为什么会展示在文档中。

image

Comment (thinkasany) [reply components/steps/style/index.ts:102]: 刚好在用ConfigProvider配置组件token，看到一处没有描述的token，然后顺便发现了这个问题 hhh

**Comment by yoyo837** [comment]
奇怪哦，哪里冒出来的image diff

**Comment by wanpan11** [comment]
> 奇怪哦，哪里冒出来的image diff

这个 diff 好像存在很久了 🤣

**Review by afc163 — APPROVED** [review]

**Pull request merged**